### DISCUSSION TITLE
How to text align to left but number align to right?

### DISCUSSION THREAD

**Discussion by kittiu** [frappe/print_designer#144] [Q&A] [answered]
This is a normal practice when print form (in fact, it was like this in ERPNext form already). Is this possible? Or at least it should be by default.

![image](https://github.com/frappe/print_designer/assets/1973598/6c35d830-3eaf-4959-8154-f7fddd80d9ca)

**Comment by maharshivpatel** [comment, marked as answer]
@kittiu this feature is not available right now. will add in some time. Thank you for sharing this issue.

**Reply by kittiu** [reply]
OK thanks!

**Comment by kittiu** [comment]
@maharshivpatel I would think this could be the most needed feature at the moment. The reason I try to use other custom code is just because this alignment.
If the number alignment is OK, other things are lower priority. :)

Thanks for the amazing product. I can say this tools save tons of time for all of us.

**Reply by maharshivpatel** [reply]
@kittiu #158 done.

**Reply by kittiu** [reply]
Amazing, thank!
